CommonTools/UtilAlgos/interface/UniqueMerger.h

```diff
@@ -14,16 +15,19 @@
  *
  * \author Lauren Hay
  *
- * \version $Revision: 1.1 $
+ * \version $Revision: 1.2 $
  *
  * 
  */
+
+#include <typeinfo>
 #include "FWCore/Framework/interface/global/EDProducer.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/Utilities/interface/transform.h"
 #include "FWCore/Utilities/interface/InputTag.h"
 #include "DataFormats/Common/interface/CloneTrait.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
```

Comment:
This include is probably not needed any more, now

Reply:
We need it back, since we have the LogWarning when enabled by a user flag.